api/client/samples/analysis_kits/stocks_to_use_price_model/stocks_to_use_price_model.ipynb

```diff
@@ -102,7 +102,7 @@
     "                                   'source_id': CME,\n",
     "                                   'frequency_id': 15,\n",
     "                                   'start_date': '2006-01-01',\n",
-    "                                   'show_revisions': True})\n",
+    "                                   'reporting_history': True})\n",
```

Comment:
should we just call "reporting_history" in API as well? ( last time I renamed it to show_reporting_date)

Reply:
discussed offline. it requires a mandatory upgrade in the client side if we want to deprecate the old param in API endpoint
leaving it as a future TODO